packages/driver/src/cy/retries.ts

```diff
@@ -73,6 +73,18 @@ export const create = (Cypress, state, timeout, clearTimeout, whenStable, finish
       }).message
 
       const retryErrProps = modifyErrMsg(error, prependMsg, (msg1, msg2) => {
+        const autOrigin = Cypress.state('autOrigin')
+        const commandOrigin = window.location.origin
```

Comment:
It looks like `autOrigin` is an origin policy and `commandOrigin` is an origin so they may differ on subdomains. Is this intentional?

Reply:
command origin is the origin of the spec bridge iframe which is set to the origin policy.

But now i'm wondering if that is true for top (when not in a spec bridge), it might not be. I will check.